src/config/doc-link.js

```diff
@@ -0,0 +1,13 @@
+const docUrl = 'https://docs.openpitrix.io';
```

Comment:
docs link is a global conf,  can be placed at `src/config/index.js`, or `server/local_config.yml`

Reply:
No need to separate `version`, `lang`

Reply:
ok